src/test/end-to-end/features/claimDetails/claimDetailsTest.js

```diff
@@ -0,0 +1,23 @@
+const { doNotHaveToCompleteCard, stepsToMakingYourClaim, claimDetails } = require('../../helpers/caseHelper');
+const commonFlow = require('../../helpers/commonFlow.js');
+const { I } = inject();
+
+Feature('Enter claim details for a claim');
+
+Scenario('Navigate Claim Details - Describe What happenned to you', async () => {
+  await commonFlow.createSingleMyselfCase();
+  await I.authenticateWithIdam();
+  await doNotHaveToCompleteCard(I);
+  await stepsToMakingYourClaim(I);
+  I.click("//a[contains(.,'Describe what happened to you')]");
+  await claimDetails(I);
+}).tag('@pats');
+
+Scenario('Navigate Claim Details - Tell us what you want from your claim', async () => {
+  await commonFlow.createSingleMyselfCase();
+  await I.authenticateWithIdam();
+  await doNotHaveToCompleteCard(I);
```

Comment:
See above, if navigating directly to the page we don't need this function

Reply:
yes the test has been modified to this effect...